dev/io.openliberty.microprofile.telemetry.internal.common/src/io/openliberty/microprofile/telemetry/internal/common/info/OpenTelemetryInfoFactoryImpl.java

```diff
@@ -185,23 +218,87 @@ private static boolean checkDisabled(Map<String, String> oTelConfigs) {
     private HashMap<String, String> getTelemetryProperties() {
         try {
             Config config = ConfigProvider.getConfig();
+
             HashMap<String, String> telemetryProperties = new HashMap<>();
             for (String propertyName : config.getPropertyNames()) {
                 if (propertyName.startsWith("otel") || propertyName.startsWith("OTEL")) {
                     String normalizedName = propertyName.toLowerCase().replace('_', '.');
+
                     config.getOptionalValue(normalizedName, String.class)
-                        .ifPresent(value -> telemetryProperties.put(normalizedName, value));
+                          .ifPresent(value -> telemetryProperties.put(normalizedName, value));
+
                 }
             }
 
-            telemetryProperties.putAll(openTelemetryVersionedConfiguration.getTelemetryPropertyDefaults());
             return telemetryProperties;
         } catch (Exception e) {
             Tr.error(tc, Tr.formatMessage(tc, "CWMOT5002.telemetry.error", e));
             return new HashMap<String, String>();
         }
     }
 
+    private HashMap<String, String> getServerTelemetryProperties() {
+        try {
+            HashMap<String, String> telemetryProperties = new HashMap<>();
+
+            AccessController.doPrivileged(new PrivilegedAction<String>() {
+                @Override
+                public String run() {
+                    Map<String, String> envProperties = System.getenv();
+                    Map<Object, Object> systemProperties = System.getProperties();
+
+                    HashMap<String, String> tempProperties = new HashMap<>();
+
+                    //Check system properties for all configured OTEL properties
+                    for (Object propertyName : systemProperties.keySet()) {
+                        String normalizedName = ((String) propertyName).toLowerCase().replace('_', '.');
+                        if (normalizedName.startsWith("otel")) {
+                            String propertyValue = (String) systemProperties.get(propertyName);
+
+                            if (propertyValue != null) {
+                                tempProperties.put(normalizedName, propertyValue);
+                            }
+                        }
+
+                    }
+
+                    //Check system environment for all configured OTEL properties and replace any
+                    //previously configured values found in the system properties.
+                    for (String propertyName : envProperties.keySet()) {
```

Comment:
System properties have a higher priority over environment variables.

Reply:
Switched around the priority. It'll now read env variables first and replace them with system properties. 

Thanks.